src/input/user.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace GOL\Input;
+
+use GOL\Boards\Board;
+use Ulrichsg\Getopt;
+
+/**
+ * User input
+ * Allows user to set cells alive manually by using the x- and y-coords of the cells.
+ * Use prepareBoard() to prepare a Board and register() to register optional arguments.
+ */
+class User extends Input
```

Comment:
So this class was implemented by Niklas?

Reply:
Yes this class was implemented by Niklass.